packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -40,7 +40,7 @@ import 'primitives/memory_utils.dart';
 // TODO(bkonyi): enable new allocation profile table when we're ready to remove
 // the existing allocations table.
 @visibleForTesting
-bool enableNewAllocationProfileTable = false;
+bool enableNewAllocationProfileTable = true;
```

Comment:
flip this back to false before submitting

Reply:
it is protected by test, so no way I can forget it :)